database/sql/schema.sql

```diff
@@ -56,10 +71,11 @@ CREATE TABLE ProfessorCoursePreference (
 	prof_relief_id BINARY(16) NOT NULL,
```

Comment:
Maybe prof_relief_id could be change to prof_avail_id or similar to keep with the name change?

Reply:
good point, I'll make that change